### PR TITLE
Pr to merge resturant page into development

### PR DESCRIPTION
**Pull request by talhawaqar**
In this pr, I used simple javascript to create the HTML of the restaurant site with following pages

1. Create a home page using javascript.
2. Create a menu page using javascript.
3. Create a contact page using javascript.
4. Create a navbar to switch between different pages using javascript

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+./node_modules
+node_modules
```
Comment (peterrobert) [inline .gitignore]: Great work ignoring the node_modules. However, you don't need two statements. One is enough. Kindly delete one

File: package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "restaurant-homepage",
+  "version": "1.0.0",
+  "description": "It is a resturant homepage which shows it's menu build by using js.",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (peterrobert) [inline package.json]: Great work adding this dependency. However, let's improve it and make it better. You can add other dependencies such as start

`"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
`

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @talhawaqar very nice progress. You have done an amazing job. Your project meets almost all the requirements 👍🏿 . Great work using es6 in your code 👏🏿 .

However, kindly fix the following three issues and submit your project for another review.

- [x]  Your pull request message should be descriptive. Kindly add a title and some points telling us what you have worked on in this milestone.


### if you have any questions kindly reach me via slack @peterrobert 

### HAPPY CODING 💻

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all requirements and getting this project approved. :+1:

You can merge this branch into the development branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Pull request merged**